arrocke/gloss-translation
# fix: verse picker mobile bug

## What has changed
- The verse picker now works on mobile. 
  - _Tested on an Android device with the Chrome browser_


## Connected Issues
closes #378 


## QA Steps 

1. Open the website on a mobile device. Here is one way of doing this (for Chrome based browsers on Android devices):
   1. Run `nx serve web` and `nx serve api` on your local device (such as a computer 
   2. Connect a USB cord from your local device to your mobile device.
   3. [Enable USB debugging](https://www.lifewire.com/enable-usb-debugging-android-4690927)
   4. Go to "chrome://inspect/#devices". Then, check the box which says "Discover USB devices": 
     <img width="646" alt="image" src="https://github.com/globalbibletools/gbt/assets/91034018/82b1a94b-9024-4ce7-a109-7c25ae54c03c">

   5. Press "Port Forwarding", add the ports 4200 and 4300, and check the box which says "Enable Port Forwarding":
      <img width="423" alt="image" src="https://github.com/globalbibletools/gbt/assets/91034018/8dd2369d-b78d-4725-8a2f-991f029e96ae">
   6. Finally, on your mobile device, navigate to "localhost:4200"

2. Type in a verse reference and hit enter. Confirm that the page navigates to the reference that you entered.


## Post-Deployment
